cmd/utils.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	airflowversions "github.com/astronomer/astro-cli/airflow_versions"
+	"github.com/astronomer/astro-cli/houston"
+	"github.com/astronomer/astro-cli/messages"
+	"github.com/pkg/errors"
+)
+
+func prepareDefaultAirflowImageTag(airflowVersion string, httpClient *airflowversions.Client, houstonClient *houston.Client, out io.Writer) (string, error) {
+	defaultImageTag, _ := airflowversions.GetDefaultImageTag(httpClient, "")
+
+	r := houston.Request{
+		Query: houston.DeploymentInfoRequest,
+	}
+
+	wsResp, err := r.DoWithClient(houstonClient)
+
+	if err == nil {
+		acceptableAirflowVersions := wsResp.Data.DeploymentConfig.AirflowVersions
+		if airflowVersion != "" && !acceptableVersion(airflowVersion, acceptableAirflowVersions) {
+			return "", errors.Errorf(messages.ERROR_INVALID_AIRFLOW_VERSION, strings.Join(acceptableAirflowVersions, ", "))
+		}
+		if airflowVersion == "" {
+			defaultImageTag = ""
+		} else {
+			defaultImageTag = fmt.Sprintf("%s-buster-onbuild", airflowVersion)
+		}
+	} else if airflowVersion != "" {
+		return "", errors.New("you can't use --airflow-version option while not authenticated to a cluster")
```

Comment:
aren't there several reasons why r.DoWithClient(houstonClient) could error? Its not always going to be b/c of not being authed